_data/patterns/race.yml

```diff
@@ -0,0 +1,88 @@
+legend: |
+  What is your race and/or ethnicity?
+hint: |
+  Select all that apply
+hint_additional: |
+  and enter additional details in the spaces below.
+categories:
```

Comment:
Wondering if we're going to alphabetize the categories. It looks like it was done in one case but not all.

Reply:
I think this will be a question for OMB - it wasn't clear to me to best to translate the order in to a single column. If you have a suggestion for re-ordering it, please let me know!

Reply:
Not sure I understand the issue you're having, but here's an example of what I'm talking about:

Here's the order under the Asian category:
Chinese
Asian Indian
Filipino
Vietnamese
Korean
Japanese

I would recommend:
Asian Indian
Chinese
Filipino
Japanese
Korean
Vietnamese

And in the "Enter, for example" area, I'd reorder from:
 Pakistani, Hmong, Afghan, etc.
 to 
Afghan, Hmong, Pakistani, etc.

I don't think it requires any new structure. It's just alphabetical order of the subcategories and examples. It hopefully removes the potential concern that one group is being listed over another. I do agree that it's a good question for OMB. Perhaps it waits for them to look at it and Dan or Anne ask the question in delivery.

Reply:
I was just assuming that the order in the provided examples was intentional. It might not be! We can leave this as an outstanding question on the PR when we pass it on to make sure it gets addressed before we publish. Are you ok if I leave it as is for now and flag it for follow-up before merge?

Reply:
I am!